Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -269,6 +269,16 @@ function SeasonOfDiscovery:LoadBaseQuests()
             [questKeys.objectivesText] = {"Return the Orb Shards to Delgren."},
             [questKeys.objectives] = nil,
         },
+        [78114] = {
+            [questKeys.name] = "Wild Wyvern Wrangling",
+            [questKeys.startedBy] = {{210845}},
+            [questKeys.finishedBy] = {{210845}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 25,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Bring a Hypnotic Crystal and a Gnarled Wand of Wild Magic to Jixo Madrocket in the Stonetalon Mountains."},
+        },
```

Comment:
```suggestion
            [questKeys.objectivesText] = {"Bring a Hypnotic Crystal and a Gnarled Wand of Wild Magic to Jixo Madrocket in the Stonetalon Mountains."},
            [questKeys.objectives] = {nil,nil,{{209838},{209840}}},
        },
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5328/commits/26f950962582fe19c980069f077a4609e04859ab